packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -180,8 +190,10 @@ export class MenusContributionPointHandler {
         const command: Command = { id };
         this.commands.registerCommand(command, handler);
 
-        const { group, when } = action;
-        const item: Mutable<TabBarToolbarItem> = { id, command: id, group: group || '', when };
+        const { when } = action;
+        const [group, sort] = (action.group || 'navigation').split('@');
```

Comment:
It'd be better to include some comments so we can easily understand what is going on here, and document what we expect.

Reply:
this code is handle group and priority
before @ is group name
after @ is priority
```
"editor/title": [
        {
          "when": "httpResponsePreviewFocus",
          "command": "rest-client.save-response",
          "group": "navigation@1"
        },
        {
          "when": "httpResponsePreviewFocus",
          "command": "rest-client.save-response-body",
          "group": "navigation@2"
        },
        {
          "when": "httpResponsePreviewFocus",
          "command": "rest-client.copy-response-body",
          "group": "navigation@3"
        },
        {
          "when": "httpResponsePreviewFocus",
          "command": "rest-client.fold-response",
          "group": "1_rest-client@1"
        },
        {
          "when": "httpResponsePreviewFocus",
          "command": "rest-client.unfold-response",
          "group": "1_rest-client@2"
        },
        {
          "when": "codeSnippetPreviewFocus",
          "command": "rest-client.copy-codesnippet",
          "group": "navigation"
        }
      ],
```

Reply:
Why do we need `~~`?